python-threatexchange/threatexchange/exchanges/clients/stopncii/api.py

```diff
@@ -62,12 +61,11 @@ class StopNCIICSPFeedback:
     tags: t.Set[str] = field(default_factory=set)  # Unstructured additional tags
     source: str = ""  # Name of the Content Service Provider (CSP)
 
-    @classmethod
-    def as_dict_for_post(cls) -> t.Dict[str, t.Any]:
```

Comment:
Weird, I'm not sure why this was a classmethod before. Clearly I never tried to write a feedback with it (hence the bug)!